pkg/model/instance.go

```diff
@@ -3,8 +3,74 @@ package model
 import (
 	"fmt"
 	"time"
+
+	"gorm.io/gorm"
 )
 
+type Deployment struct {
+	ID        uint      `json:"id" gorm:"primaryKey"`
+	CreatedAt time.Time `json:"createdAt"`
+	UpdatedAt time.Time `json:"updatedAt"`
+
+	UserID uint  `json:"userId"`
+	User   *User `json:"user,omitempty"`
+
+	Name        string `json:"name" gorm:"index:idx_name_and_group,unique"`
+	Description string `json:"description"`
+	GroupName   string `json:"groupName" gorm:"index:idx_name_and_group,unique; references:Name"`
+	Group       *Group `json:"group,omitempty"`
+	TTL         uint   `json:"ttl"`
+
+	Instances []*DeploymentInstance `json:"instances"`
+}
+
+type Parameters map[string]DeploymentInstanceParameter
```

Comment:
Since all our models are in the same package I would prefix this with `Deployment` or `DeploymentInstance`. To make it clear when using the package `model.StackParameter`, `model.DeploymentParameter`.